dask/bytes/s3.py

```diff
@@ -64,30 +100,32 @@ def read_bytes(path, s3=None, delimiter=None, not_zero=False, blocksize=2**27,
 
         token = tokenize(info['ETag'], delimiter, blocksize, not_zero, compression)
 
-        logger.debug("Read %d blocks of binary bytes from %s", len(offsets), path)
+        s3_storage_options = s3.get_delegated_s3pars()
```

Comment:
Agreed. When including credentials in the URI, we should be using a delegate token to avoid sending those raw credentials across the wire.